backend/tournesol/models/entity.py

```diff
@@ -25,51 +25,97 @@
 CRITERIAS = settings.CRITERIAS
 
 
-class Video(models.Model, WithFeatures, WithEmbedding):
-    """One video."""
+class Entity(models.Model, WithFeatures, WithEmbedding):
+    """
+    A generic entity that can be compared with another one.
 
+    The current model still contains fields from the previous `Video` model.
+    These fields are kept as-is for now to ease the refactor of the Tournesol
+    app, and will be replaced in the future by the `metadata` JSON field.
```

Comment:
How about leaving `TODO` formatted comments to simplify coming back for clean-up where needed?

Edited: héhé, I see you did this a lot below. 👌

Reply:
Yup, I've also regrouped all fields inside to-do blocks to avoid repeating the to-do comment.